ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -66,7 +66,7 @@ readonly BAZEL_CACHE="https://storage.googleapis.com/cloud-cpp-bazel-cache"
 # not hit this cache.
 if [[ -r "${GOOGLE_APPLICATION_CREDENTIALS}" ]]; then
   io::log "Using bazel remote cache: ${BAZEL_CACHE}"
-  bazel_args+=("--remote_cache=${BAZEL_CACHE}")
+  bazel_args+=("--remote_cache=https://storage.googleapis.com/cloud-cpp-bazel-cache/docker/${BUILD_NAME}")
```

Comment:
Did you want to use `${BAZEL_CACHE}` here rather than retyping part of the URL string?

Reply:
Fixed.